apps/data-scanner/lib/db/migrations/sqls/20220410070449-add-table-sessions-up.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE users
+RENAME COLUMN user_id TO id;
+
+ALTER TABLE tracked_phones
+ADD COLUMN id SERIAL PRIMARY KEY,
+ADD COLUMN session_id NUMERIC,
```

Comment:
Using id here is not a good idea. A thought that we agreed on using a phone number as an identifier.